components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -175,21 +182,18 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
       this.store.select(teamUsers),
       this.store.select(getUsersStatus)]).pipe(
         takeUntil(this.isDestroyed),
-        filter(([_allUsers, uStatus, _teamUsers, tuStatus]:
-          [User[], EntityStatus, string[], EntityStatus]) =>
+        filter(([_allUsers, uStatus, _teamUsers, tuStatus]) =>
             uStatus === EntityStatus.loadingSuccess &&
             tuStatus === EntityStatus.loadingSuccess),
```

Comment:
Might want to introduce the same `loading$` Observable in this file as in the others, which would (a) allow using that same loading call (`map((statuses: EntityStatus[]) => ...) and then would simplify this block a bit. (But it is working as is, so feel free to ignore 🤷‍♂ ).

Reply:
Yeah, I did pull out the loading variable for this part once. I reverted the change because there is a `this.isLoadingTeam` variable that is used for the page instead. I was confused enough from that to leave that section alone. 
https://github.com/chef/automate/blob/4e150ee6da4c9203f6ff5a1b8339bcf6e23335c5/components/automate-ui/src/app/pages/team-details/team-details.component.ts#L118-L121